src/OrderFormAcceptanceTests.TestData/Helpers/OrderItemHelper.cs

```diff
@@ -70,7 +70,7 @@ public static async Task<OrderItem> AddRecipientToOrderItem(OrderItem orderItem,
         public static async Task<int> GetNumberOfPricingUnitsForItemAsync(string itemId, string connectionString)
         {
             var query = @"SELECT COUNT(*)
-                          FROM [buyingcatalogue].[dbo].[CataloguePrice]
+                          FROM CataloguePrice
```

Comment:
Sorry, probably should have mentioned that it's worth keeping the schema name `dbo`.